tests/MoneyTest.php

```diff
@@ -431,11 +431,12 @@ public function providerAllocateWithRemainder() : array
             [['99.99', 'USD'], [100, 100], ['USD 49.99', 'USD 49.99', 'USD 0.01']],
             [[100, 'USD'], [30, 20, 40], ['USD 33.33', 'USD 22.22', 'USD 44.44', 'USD 0.01']],
             [[100, 'USD'], [30, 20, 40, 40], ['USD 23.07', 'USD 15.38', 'USD 30.76', 'USD 30.76', 'USD 0.03']],
-            [[100, 'CHF', new CashContext(5)], [1, 2, 3, 7], ['CHF 7.65', 'CHF 15.35', 'CHF 23.05', 'CHF 53.80', 'CHF 0.15']],
+            [[100, 'CHF', new CashContext(5)], [1, 2, 3, 7], ['CHF 7.65', 'CHF 15.30', 'CHF 22.95', 'CHF 53.55', 'CHF 0.55']],
```

Comment:
i can't see how this works.
why is the remainder 55? why can't this be split out further more to the groups?

Reply:
Ah yes. Thanks.

Reply:
I still don't understand why this test change is "better"

1(100/13) = 7.6923076923 = 7.65
2(100/13) = 15.3846153846 = 15.35
3(100/13) = 23.0769230769 = 23.05
7(100/13) = 53.8461538462 = 53.80

leaving 0.15 change.
 why are the new test values better @BenMorel ?

Reply:
makes sense again, thanks!

Reply:
Thanks for explaining, @artfulrobot ;)